### PR TITLE
feat(Atoms): migrate all Atom stories

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Migrates all the atoms to the new storybook version

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [GM-840]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[GM-840]: https://skillsoftdev.atlassian.net/browse/GM-840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672ceaf0949be5d8af3958ba--gamut-preview.netlify.app>https://672ceaf0949be5d8af3958ba--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672ceaf0949be5d8af3958ba>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.b6d6ef.0<br/>@codecademy/gamut@57.4.5-alpha.b6d6ef.0<br/>@codecademy/gamut-icons@9.32.2-alpha.b6d6ef.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.b6d6ef.0<br/>@codecademy/gamut-kit@0.6.453-alpha.b6d6ef.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.b6d6ef.0<br/>@codecademy/gamut-styles@17.1.2-alpha.b6d6ef.0<br/>@codecademy/gamut-tests@5.1.4-alpha.b6d6ef.0<br/>@codecademy/macros@3.0.5-alpha.b6d6ef.0<br/>@codecademy/styleguide@68.1.4-alpha.b6d6ef.0<br/>@codecademy/variance@0.21.5-alpha.b6d6ef.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.34ceb2.0<br/>@codecademy/gamut@57.4.5-alpha.34ceb2.0<br/>@codecademy/gamut-icons@9.32.2-alpha.34ceb2.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.34ceb2.0<br/>@codecademy/gamut-kit@0.6.453-alpha.34ceb2.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.34ceb2.0<br/>@codecademy/gamut-styles@17.1.2-alpha.34ceb2.0<br/>@codecademy/gamut-tests@5.1.4-alpha.34ceb2.0<br/>@codecademy/macros@3.0.5-alpha.34ceb2.0<br/>@codecademy/styleguide@68.1.4-alpha.34ceb2.0<br/>@codecademy/variance@0.21.5-alpha.34ceb2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://672d215003fe1b4958b828bf--gamut-preview.netlify.app>https://672d215003fe1b4958b828bf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/672d215003fe1b4958b828bf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.cd422a.0<br/>@codecademy/gamut@57.4.5-alpha.cd422a.0<br/>@codecademy/gamut-icons@9.32.2-alpha.cd422a.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.cd422a.0<br/>@codecademy/gamut-kit@0.6.453-alpha.cd422a.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.cd422a.0<br/>@codecademy/gamut-styles@17.1.2-alpha.cd422a.0<br/>@codecademy/gamut-tests@5.1.4-alpha.cd422a.0<br/>@codecademy/macros@3.0.5-alpha.cd422a.0<br/>@codecademy/styleguide@68.1.4-alpha.cd422a.0<br/>@codecademy/variance@0.21.5-alpha.cd422a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://673390261b0147154396385b--gamut-preview.netlify.app>https://673390261b0147154396385b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/673390261b0147154396385b>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Atoms/FocusTrap/FocusTrap.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+
+import { ComponentHeader } from '~styleguide/blocks';
+
+import * as FocusTrapStories from './FocusTrap.stories';
+
+export const parameters = {
+  subtitle: "Used for modal and dialog experiences like `Overlay` and `ModalDeprecated`, this component controls focus management behavior when we need to limit focus to a single area of the page for some amount of time. It doesn't need to be a modal or overlay, and can instead be controlled with the `active` prop.",
+  status: 'current',
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/FocusTrap/FocusTrap.mdx:9]: I know you're gonna go and fix these `parameters` props — but wanted to note that in Cass's PR there's also a source prop now too (sorry me linking skills are bad... https://github.com/Codecademy/gamut/pull/2961/files (and in the template)

File: packages/styleguide/src/lib/Atoms/Illustrations/Illustrations.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import * as illustrations from '@codecademy/gamut-illustrations';
+import { Canvas, Controls, IconGallery, IconItem, Meta } from '@storybook/blocks';
+
+import { ComponentHeader } from '~styleguide/blocks';
+
+import * as IllustrationsStories from './Illustrations.stories';
+
+export const parameters = {
+  subtitle: "We're developing a set of fun illustrations given us to design",
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/Illustrations/Illustrations.mdx:9]: Nit: this is carried over from the current gamut page, but this sentence doesn't make sense to me 
It's like missing a word or two? 
Maybe: We're developing a set of fun illustrations used for design? 
But the subtitle could be more useful — "React components that serve up SVGs to provide more control and reusability."

Comment (aresnik11) [reply packages/styleguide/src/lib/Atoms/Illustrations/Illustrations.mdx:9]: lol

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.b3ffd8.0<br/>@codecademy/gamut@57.4.5-alpha.b3ffd8.0<br/>@codecademy/gamut-icons@9.32.2-alpha.b3ffd8.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.b3ffd8.0<br/>@codecademy/gamut-kit@0.6.453-alpha.b3ffd8.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.b3ffd8.0<br/>@codecademy/gamut-styles@17.1.2-alpha.b3ffd8.0<br/>@codecademy/gamut-tests@5.1.4-alpha.b3ffd8.0<br/>@codecademy/macros@3.0.5-alpha.b3ffd8.0<br/>@codecademy/styleguide@68.1.4-alpha.b3ffd8.0<br/>@codecademy/variance@0.21.5-alpha.b3ffd8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6736631b65254e64ba59b17b--gamut-preview.netlify.app>https://6736631b65254e64ba59b17b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6736631b65254e64ba59b17b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.fb11f2.0<br/>@codecademy/gamut@57.4.5-alpha.fb11f2.0<br/>@codecademy/gamut-icons@9.32.2-alpha.fb11f2.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.fb11f2.0<br/>@codecademy/gamut-kit@0.6.453-alpha.fb11f2.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.fb11f2.0<br/>@codecademy/gamut-styles@17.1.2-alpha.fb11f2.0<br/>@codecademy/gamut-tests@5.1.4-alpha.fb11f2.0<br/>@codecademy/macros@3.0.5-alpha.fb11f2.0<br/>@codecademy/styleguide@68.1.4-alpha.fb11f2.0<br/>@codecademy/variance@0.21.5-alpha.fb11f2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://673b93dfce47bb08f2b3044a--gamut-preview.netlify.app>https://673b93dfce47bb08f2b3044a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/673b93dfce47bb08f2b3044a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.13609a.0<br/>@codecademy/gamut@57.4.5-alpha.13609a.0<br/>@codecademy/gamut-icons@9.32.2-alpha.13609a.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.13609a.0<br/>@codecademy/gamut-kit@0.6.453-alpha.13609a.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.13609a.0<br/>@codecademy/gamut-styles@17.1.2-alpha.13609a.0<br/>@codecademy/gamut-tests@5.1.4-alpha.13609a.0<br/>@codecademy/macros@3.0.5-alpha.13609a.0<br/>@codecademy/styleguide@68.1.4-alpha.13609a.0<br/>@codecademy/variance@0.21.5-alpha.13609a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6740f76cffada15737b920b7--gamut-preview.netlify.app>https://6740f76cffada15737b920b7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6740f76cffada15737b920b7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.f82a6d.0<br/>@codecademy/gamut@57.5.2-alpha.f82a6d.0<br/>@codecademy/gamut-icons@9.34.1-alpha.f82a6d.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.f82a6d.0<br/>@codecademy/gamut-kit@0.6.455-alpha.f82a6d.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.f82a6d.0<br/>@codecademy/gamut-styles@17.1.2-alpha.f82a6d.0<br/>@codecademy/gamut-tests@5.1.4-alpha.f82a6d.0<br/>@codecademy/macros@3.0.5-alpha.f82a6d.0<br/>@codecademy/styleguide@68.1.4-alpha.f82a6d.0<br/>@codecademy/variance@0.21.5-alpha.f82a6d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6745f1613344dd969fcceead--gamut-preview.netlify.app>https://6745f1613344dd969fcceead--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6745f1613344dd969fcceead>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://674f391307197f42cc44b37b--gamut-preview.netlify.app>https://674f391307197f42cc44b37b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/674f391307197f42cc44b37b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.d39679.0<br/>@codecademy/gamut@57.5.2-alpha.d39679.0<br/>@codecademy/gamut-icons@9.34.1-alpha.d39679.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.d39679.0<br/>@codecademy/gamut-kit@0.6.455-alpha.d39679.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.d39679.0<br/>@codecademy/gamut-styles@17.1.2-alpha.d39679.0<br/>@codecademy/gamut-tests@5.1.4-alpha.d39679.0<br/>@codecademy/macros@3.0.5-alpha.d39679.0<br/>@codecademy/styleguide@68.1.4-alpha.d39679.0<br/>@codecademy/variance@0.21.5-alpha.d39679.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67531282665056781f88c0d8--gamut-preview.netlify.app>https://67531282665056781f88c0d8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67531282665056781f88c0d8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.a67366.0<br/>@codecademy/gamut@57.5.2-alpha.a67366.0<br/>@codecademy/gamut-icons@9.34.1-alpha.a67366.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.a67366.0<br/>@codecademy/gamut-kit@0.6.455-alpha.a67366.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.a67366.0<br/>@codecademy/gamut-styles@17.1.2-alpha.a67366.0<br/>@codecademy/gamut-tests@5.1.4-alpha.a67366.0<br/>@codecademy/macros@3.0.5-alpha.a67366.0<br/>@codecademy/styleguide@68.2.0-alpha.a67366.0<br/>@codecademy/variance@0.21.5-alpha.a67366.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6753424b59982521a34b821e--gamut-preview.netlify.app>https://6753424b59982521a34b821e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6753424b59982521a34b821e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.5991e7.0<br/>@codecademy/gamut@57.5.3-alpha.5991e7.0<br/>@codecademy/gamut-icons@9.34.2-alpha.5991e7.0<br/>@codecademy/gamut-illustrations@0.49.3-alpha.5991e7.0<br/>@codecademy/gamut-kit@0.6.456-alpha.5991e7.0<br/>@codecademy/gamut-patterns@0.9.18-alpha.5991e7.0<br/>@codecademy/gamut-styles@17.1.3-alpha.5991e7.0<br/>@codecademy/gamut-tests@5.1.5-alpha.5991e7.0<br/>@codecademy/macros@3.0.5-alpha.5991e7.0<br/>@codecademy/styleguide@68.2.0-alpha.5991e7.0<br/>@codecademy/variance@0.21.5-alpha.5991e7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6753451ec1b81529c63cab34--gamut-preview.netlify.app>https://6753451ec1b81529c63cab34--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6753451ec1b81529c63cab34>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Atoms/Card/Card.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+<Canvas of={CardStories.OutlineShadow} />
+<Canvas of={CardStories.OutlineShadowColor} />
+
+## As link
+
+<Canvas of={CardStories.Link} />
+
+## ColorMode as default
+
+Card elements respond to the current **ColorMode** [TODO: link to colormode page] they are used in. Note that their background color will match the background of their context and secondary colors are used for their hover effect. As these are not the typical inverse effects we utilize in color mode, please check with your design team before use.
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/Card/Card.mdx:75]: Leftover "TODO" to include a LinkTo

File: packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+There are 2 types of shadow directions. The default shadow offset is bottom left.
+
+<Canvas of={FloatingCardStories.ShadowBottomLeft} />
+<Canvas of={FloatingCardStories.ShadowBottomRight} />
+
+## Shadow pattern
+
+We can specify the pattern of the FloatingCard shadow by providing the `pattern` prop with the imported pattern from `@codecademy/gamut-patterns`.
+
+{/* See the <LinkTo component="Atoms/Patterns">`Patterns`</LinkTo> story to view all the possible pattern options. */}
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx:52]: I think you can remove the commenting around this sentence now

File: packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+<Canvas of={FloatingCardStories.ShadowPattern} />
+
+## Card wrapper display
+
+The wrapper display may need to be changed from the default `inline-block`
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx:58]: This isn't your thing lol but I think this sentence can be touched up — 
Depending on the accuracy, something like:

By default a `FloatingCard` has a wrapper with a display of `inline-block`. However, if you need to you can set the `cardDisplay` to `block`, as shown below: 

But also seems like `cardDisplay` might be `containerDisplay`? 
https://github.com/Codecademy/gamut/blob/main/packages/gamut/src/FloatingCard/FloatingCard.tsx 
see line 95

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67535138a37b223dc41a2184--gamut-preview.netlify.app>https://67535138a37b223dc41a2184--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67535138a37b223dc41a2184>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Atoms/FormInputs/Input/Input.mdx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+<Canvas of={InputStories.Disabled} />
+
+## Custom icons
+
+You can use a custom icon (from gamut-icons) in your Input elements. The error and valid icons will overwrite the custom icon if those states are applied to the element.
+
+<Canvas of={InputStories.CustomIcon} />
+
+## FormGroup + Input
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/FormInputs/Input/Input.mdx:67]: Would be nice to add some context, something like:
Input elements can be wrapped in a <LinkTo id="atoms-formelements--formgroup">`FormGroup` component</LinkTo> to provide a label. The label can also be customized to your needs as well. 

^ not sure if that's the right id fwiw lol

File: packages/styleguide/src/lib/Atoms/FormInputs/Select/Select.mdx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+### Disabled
+
+<Canvas of={SelectStories.Disabled} />
+
+### Error
+
+<Canvas of={SelectStories.Error} />
+
+## FormGroup + Select
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/FormInputs/Select/Select.mdx:49]: Similar feedback to input, consider adding:  

Input elements can be wrapped in a FormGroup component to provide a label. The label can also be customized to your needs as well.

^ still not sure if that's the right id fwiw :)

File: packages/styleguide/src/lib/Atoms/Icons/Regular/Regular.mdx [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+## General infomation
+
+For regular icons, we use a subset of [Streamline's "Ultimate Regular" icons](https://www.streamlinehq.com/icons/streamline-regular).
+
+Our icons are standardized with a viewBox size of `"0 0 24 24"`. Icons should be centered and take up the entirety of this viewBox.
+
+All of our Icons are not minified by default, and optimized by [SVGO](https://github.com/svg/svgo) before build time. We like to preserve the source URL as well as any relevent comments or metadata within these SVGs.
+
+## Adding an icon
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/Icons/Regular/Regular.mdx:39]: This is my mistake, I added the instructions in the About page and didn't delete this... could you remove this section? Please and thank you 🙏

File: packages/styleguide/src/lib/Atoms/Loaders/About.mdx [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+<AboutHeader {...parameters} />
+
+<TableOfContents
+  links={[
+    {
+      id: 'Atoms/Loaders/Shimmer',
+      subtitle: 'Shimmer effect for boxes — generic styles that apply to our Skeleton elements.',
+      title: 'Shimmer',
+      status: 'current',
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Atoms/Loaders/About.mdx:21]: The story for shimmer says "updating"

**Review by LinKCoding — CHANGES_REQUESTED** [review]
Left some thoughts, but looks good overall!!

File: packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx (hunk 4 above)
Comment (aresnik11) [reply packages/styleguide/src/lib/Atoms/FloatingCard/FloatingCard.mdx:52]: yeppp thanks

File: packages/styleguide/src/lib/Atoms/FormInputs/Input/Input.mdx (hunk 6 above)
Comment (aresnik11) [reply packages/styleguide/src/lib/Atoms/FormInputs/Input/Input.mdx:67]: i went with some existing copy within SelectDropdown and added a link to FormGroup
>`SelectDropdown` formats nicely in FormGroups and reflects error states. Make sure you provide an htmlFor to the `FormGroup` and a `name` to the `SelectDropdown` for maximum accessibility. See <LinkTo id="Atoms/FormElements/FormGroup">FormGroup</LinkTo> story for the customizations available.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.15ec61.0<br/>@codecademy/gamut@57.5.3-alpha.15ec61.0<br/>@codecademy/gamut-icons@9.34.2-alpha.15ec61.0<br/>@codecademy/gamut-illustrations@0.49.3-alpha.15ec61.0<br/>@codecademy/gamut-kit@0.6.456-alpha.15ec61.0<br/>@codecademy/gamut-patterns@0.9.18-alpha.15ec61.0<br/>@codecademy/gamut-styles@17.1.3-alpha.15ec61.0<br/>@codecademy/gamut-tests@5.1.5-alpha.15ec61.0<br/>@codecademy/macros@3.0.5-alpha.15ec61.0<br/>@codecademy/styleguide@68.2.0-alpha.15ec61.0<br/>@codecademy/variance@0.21.5-alpha.15ec61.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67537008613d0022501a2b34--gamut-preview.netlify.app>https://67537008613d0022501a2b34--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67537008613d0022501a2b34>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Atoms/Drawer/Drawer.mdx [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+## Usage
+
+Drawers animate between having `0` width and having a responsive width:
+
+- Mobile devices (&lt;`sm`): `75vw`
+- Tablet and larger (&gt;=`sm`): `30rem`
+
+{/* An example of a molecule that uses Drawer is the <LinkTo component="Molecules/Flyout">Flyout</LinkTo>. */}
+
```
[the remark this answers is not in the document]
Comment (aresnik11) [reply packages/styleguide/src/lib/Atoms/Drawer/Drawer.mdx:34]: thanks

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.c77120.0<br/>@codecademy/gamut@57.5.3-alpha.c77120.0<br/>@codecademy/gamut-icons@9.34.2-alpha.c77120.0<br/>@codecademy/gamut-illustrations@0.49.3-alpha.c77120.0<br/>@codecademy/gamut-kit@0.6.456-alpha.c77120.0<br/>@codecademy/gamut-patterns@0.9.18-alpha.c77120.0<br/>@codecademy/gamut-styles@17.1.3-alpha.c77120.0<br/>@codecademy/gamut-tests@5.1.5-alpha.c77120.0<br/>@codecademy/macros@3.0.5-alpha.c77120.0<br/>@codecademy/styleguide@68.2.0-alpha.c77120.0<br/>@codecademy/variance@0.21.5-alpha.c77120.0<br/></pre>

**Review by LinKCoding — APPROVED** [review]
🔥⚛️🔥⚛️ 🔥⚛️🔥

**Pull request merged**